redpanda/templates/statefulset.yaml

```diff
@@ -70,6 +70,9 @@ spec:
               CONFIG=/etc/redpanda/redpanda.yaml;
               NODE_ID=${SERVICE_NAME##*-};
               cp /tmp/base-config/redpanda.yaml "$CONFIG";
+              {{- if (include "redpanda.semver" . | semverCompare ">=22.1.4") }}
```

Comment:
Same as previous comments: semver should be `22.1.1`

Reply:
Fixed in [519e4f1](https://github.com/redpanda-data/helm-charts/pull/117/commits/519e4f193b60c4e2410f2b1042c3eb2d958f5ea7)